tests/foreman/ui_airgun/test_host.py

```diff
@@ -223,3 +306,180 @@ def test_positive_inherit_puppet_env_from_host_group_when_action(session):
         values = session.host.read(host.name)
         assert values['host']['hostgroup'] == hostgroup.name
         assert values['host']['puppet_environment'] == env.name
+
+
+@tier3
```

Comment:
i'm not sure why this (and all the others) are marked as tier3 - the host is fake, i would go with t2